### PR TITLE
Event Dashboard: Improve view of events

### LINKED ISSUE #1993: Event Dashboard: Improve view of events
The new event dashboard needs some improvements (https://next.eventyay.com/events/live). Make it look more similar to the version 1 please e.g.:
* Align on top
* Make tickets and its info in one row.
* Distribute columns more efficiently

screenshot from 2019-01-27 06-34-57

### PR DESCRIPTION
**Pull request by harsh-jindal**
#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Event Dashboard: Improve view of events

screenshot 2019-01-28 at 9 30 34 pm

#### Changes proposed in this pull request:

-Aligned on top
-Made tickets and its info in one row.
-Numbered the user role

Fixes #1993

### CODE REVIEW

**Comment by harsh-jindal** [comment]
@CosmicCoder96 Please review

File: app/templates/components/ui-table.hbs [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {{#if isLoading}}
   <div class="ui active inverted dimmer">
-    <div class="ui small text loader">Loading</div>
+    <div class="ui small text loader">
+      Loading
+    </div>
```
Comment (uds5501) [inline app/templates/components/ui-table.hbs]: Just a query, what does this really change?

File: app/templates/components/ui-table.hbs [hunk 2]
```diff
@@ -15,7 +17,14 @@
...
       <table class="{{themeInstance.classes.table}}">
-        <thead class="{{if noHeaderFilteringAndSorting 'table-header-no-filtering-and-sorting'}} {{themeInstance.classes.thead}}">
+        <thead
+          class="{{if
+              noHeaderFilteringAndSorting
+              "table-header-no-filtering-and-sorting"
+            }}
+
+            {{themeInstance.classes.thead}}"
+        >
```
Comment (uds5501) [inline app/templates/components/ui-table.hbs]: Same question here, what was wrong in the initial format as long as it was not encountering any template linting errors?
What was the motivation behind changing it?

File: app/templates/components/ui-table/cell/cell-sessions-dashboard.hbs [hunk 3]
```diff
@@ -1,8 +1,8 @@
...
-  <div class="item">{{t 'Accepted'}}: {{record.eventStatisticsGeneral.sessionsAccepted}}</div>
-  <div class="item">{{t 'Confirmed'}}: {{record.eventStatisticsGeneral.sessionsConfirmed}}</div>
-  <div class="item">{{t 'Pending'}}: {{record.eventStatisticsGeneral.sessionsPending}}</div>
-  <div class="item">{{t 'Rejected'}}: {{record.eventStatisticsGeneral.sessionsRejected}}</div>
+  <div class="item">1. {{t 'Draft'}}: {{record.eventStatisticsGeneral.sessionsDraft}}</div>
+  <div class="item">2. {{t 'Submitted'}}: {{record.eventStatisticsGeneral.sessionsSubmitted}}</div>
+  <div class="item">3. {{t 'Accepted'}}: {{record.eventStatisticsGeneral.sessionsAccepted}}</div>
+  <div class="item">4. {{t 'Confirmed'}}: {{record.eventStatisticsGeneral.sessionsConfirmed}}</div>
+  <div class="item">5. {{t 'Pending'}}: {{record.eventStatisticsGeneral.sessionsPending}}</div>
+  <div class="item">6. {{t 'Rejected'}}: {{record.eventStatisticsGeneral.sessionsRejected}}</div>
```
Comment (uds5501) [inline app/templates/components/ui-table/cell/cell-sessions-dashboard.hbs]: @harsh-jindal Kindly refer to this issue #1809 . This particular file change is simply reintroducing the potential space wastage and dis-alignment. So i'd suggest that you remove the numbering

File: app/templates/components/ui-table.hbs (hunk 1 above)
Comment (harsh-jindal) [reply app/templates/components/ui-table.hbs]: @uds5501 I guess my text editor formatted that automatically. Will correct it.

File: app/templates/components/ui-table.hbs (hunk 2 above)
Comment (harsh-jindal) [reply app/templates/components/ui-table.hbs]: @uds5501 I guess my text editor formatted that automatically. Will correct it.

File: app/templates/components/ui-table/cell/cell-sessions-dashboard.hbs (hunk 3 above)
Comment (harsh-jindal) [reply app/templates/components/ui-table/cell/cell-sessions-dashboard.hbs]: @uds5501 I'll make the changes.

File: app/helpers/list-item.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import { helper } from '@ember/component/helper';
```
Comment (niranjan94) [inline app/helpers/list-item.js]: Why do you need a new helper for this ? Ember math helpers are already part of the project. Use that to increment the index by 1 and display. Remove this one .

**Comment by harsh-jindal** [comment]
@niranjan94 Please review.

**Pull request merged**